bin/qbatch

```diff
@@ -30,7 +30,10 @@ OPTIONS = [os.environ.get("QBATCH_OPTIONS")] if os.environ.get(
     "QBATCH_OPTIONS") else []
 
 if os.getenv('SGE_ROOT'):
-    SYSTEM = os.environ.get("QBATCH_SYSTEM", "sge")
+    if os.environ.get("QBATCH_SYSTEM", "sge") or os.environ.get("QBATCH_SYSTEM", "slurm"):
+        SYSTEM = os.environ.get("QBATCH_SYSTEM")
+    else:
+        sys.exit("specify either sge or slurm for QBATCH_SYSTEM")
```

Comment:
Thanks for looking into it. You're right. I'm also getting familiar with slurm, and the issue at the moment is that I couldn't find slurm environment variables on the login node. They appear to be set once we are on a compute node. I could just replace that with a test:
`if which("sbatch"): ...`